coalib/settings/Section.py

```diff
@@ -161,6 +164,13 @@ def __getitem__(self, item, ignore_defaults=False):
 
         res = self.contents.get(key, None)
         if res is not None:
+            if res.to_append:
+                if self.defaults:
+                    if res.key in self.defaults:
+                        default_val = self.defaults[res.key].value
+                        self.update_setting(
+                            res.key, new_value=default_val + ',' + res.value)
+                        res.to_append = False
```

Comment:
I really don't like this idea of creating a new setting when you get it. It feels like the setting might need a reference to the default setting or something that should be set upon setting the default section or so... I'm not fully sure but this doesn't feel clean